.github/workflows/back-to-development.yaml

```diff
@@ -0,0 +1,54 @@
+name: Go back to Development
+on:
+  workflow_dispatch:
```

Comment:
If this fails, it can be restarted with manual trigger. However, no input is needed because in a later step, version is read from bufcli.go